manifests/profile/haproxy.pp

```diff
@@ -114,4 +114,6 @@
     tag    => 'haproxy'
   }
 
+  nebula::haproxy::kubernetes_cluster { 'just open port 6443 for now': }
```

Comment:
Why not just put the firewall rules right here?

Reply:
My thinking was that we'd need a defined type eventually, so I just went ahead and made one—but looking back I think that for now it'd be better to do this the simpler way and just put the firewall rules here, yeah.